### ISSUE TITLE
Readme needs some serious updates

### ISSUE THREAD

**Issue by PtrMan** [opennars/opennars#214]
* introduction need a new line after the logo
* update project structure

**Comment by TonyLo1** [comment]
I agree - several related issues raised already - I am consolidating and will resolve the first pass issue tomorrow. Such as old links etc - The bigger issue of 'artistic impression' I will start tomorrow but will need input from all.
